src/Assets/LightSkyDefense/Scripts/Path/Path.cs

```diff
@@ -12,6 +12,9 @@ public class Path : MonoBehaviour
     [Tooltip("Amount of points to generate that form the line")]
     public int WaypointCount = 100;
 
+    [Tooltip("The radius that the capsule colliders in the path will get")]
+    private float PathWidth = 0.15f;
```

Comment:
_pathWidth

Reply:
I'm not going to change the name since this field should be public.